client/scss/utilities/variables_config/colors.json

```diff
@@ -0,0 +1,30 @@
+{
+  "'seaweed'": ["#034e44", "primary"],
+  "'mint'": ["#0aa38f", "primary"],
+  "'white'": ["#ffffff", "primary"],
+  "'cream'": ["#f0ede3", "primary"],
+  "'black'": ["#1d1d1d", "primary"],
+  "'dark-black'": ["#010101", "primary"],
+  "'turquoise-graphics'": ["#5cb8bf", "secondary"],
+  "'turquoise-text'": ["#367378", "secondary"],
+  "'teal'": ["#006272", "secondary"],
+  "'petrol'": ["#00424d", "secondary"],
+  "'purple-graphics'": ["#b67bc2", "secondary"],
+  "'purple-text'": ["#895791", "secondary"],
+  "'indigo'": ["#46284d", "secondary"],
+  "'red-graphics'": ["#ff4355", "secondary"],
+  "'red-text'": ["#c72e3d", "secondary"],
+  "'orange-graphics'": ["#e87500", "secondary"],
+  "'orange-text'": ["#ad4e00", "secondary"],
+  "'yellow'": ["#ffce3c", "secondary"],
+  "'pewter'": ["#717171", "secondary"],
+  "'silver'": ["#8f8f8f", "secondary"],
+  "'smoke'": ["#e8e8e8", "secondary"],
+  "'pumice'": ["#d9d6ce", "secondary"],
+  "'charcoal'": ["#323232", "secondary"],
+  "'java'": ["#1fbea9", "secondary"],
+  "'cotton-seed'": ["#bcbab5", "secondary"],
+  "'transparent'": ["transparent", "tertiary"],
+  "'inherit'": ["inherit", "tertiary"],
+  "'currentColor'": ["currentColor", "tertiary"]
```

Comment:
Yes – hard to see how to do this using [https://www.npmjs.com/package/json-sass](json-sass) though.